packages/realm-server/tests/headless-chrome-indexing-test.ts

```diff
@@ -1477,19 +1646,6 @@ module(basename(__filename), function () {
           }
         `,
       );
-      assert.deepEqual(
-        { ...realm.realmIndexUpdater.stats },
-        {
-          instancesIndexed: 0,
-          instanceErrors: 2,
-          moduleErrors: 2,
-          modulesIndexed: 0,
-          definitionErrors: 0,
-          definitionsIndexed: 0,
-          totalIndexEntries: 20,
-        },
-        'indexed correct number of files',
-      );
```

Comment:
in general i'm starting to move away from these types of assertions. they are too brittle, and often making assertions around counts that have nothing to do with the tests. there are other more robust ways to assert that an instance was indexed (like make a change to it and search for that change--which we do in a lot of places that we are also making this kind of assertion)